### ISSUE TITLE
Change ports for external streaming at the same time

### ISSUE THREAD

**Issue by dbwns373** [moonlight-stream/moonlight-qt#1092]
There are two computers in one network
You can stream both devices simultaneously over Wi-Fi from the inside,

Port forwarding is coming into the same ports on different internal IPs (two computers), 
so you can't stream both ports simultaneously if they're connected from outside
Is there an option to change ports or a way to stream two computers from outside at the same time?

**Comment by jorys-paulin** [comment]
Yes, but it's only possible with Sunshine, GeForce Experience uses fixed ports.
You can specify which ports Sunshine should use by following the instructions here: https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/advanced_usage.html#port.
You'll have to forward those new ports if Sunshine can't use UPnP to do it for you.
Then to add that computer in Moonlight, you can manually enter your IP address and the "config" port at the end, ie `1.1.1.1:1234`.
